rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -180,7 +185,7 @@ def train(
 
         batch_size = 64
 
-        for attribute in DENSE_FEATURIZABLE_ATTRIBUTES:
+        for attribute in tqdm(DENSE_FEATURIZABLE_ATTRIBUTES):
```

Comment:
I don't think this is the right place for tqdm. I think it should be over train examples